downstream/assemblies/platform/assembly-installing-controller-operator.adoc

```diff
@@ -8,14 +8,14 @@ ifdef::context[:parent-context: {context}]
 
 
 [id="installing-controller-operator"]
-= Installing {ControllerName} on {OCP} web console
+= Installing and configuring {ControllerName} on {OCP} web console
 
 
 :context: installing-contr-operator
 
 
 [role="_abstract"]
-You can use these instructions to install the {ControllerName} operator on {OCP}, as well as specify customer resources.
+You can use these instructions to install the {ControllerName} operator on {OCP}, specify custom resources, as well as deploying {PlatformNameShort} with an external database.
```

Comment:
Suggestion: `You can use these instructions to install the {ControllerName} operator on {OCP}, specify custom resources, and deploy {PlatformNameShort} with an external database.`